src/akkudoktoreos/prediction/self_consumption_probability.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
The use of underscores in Python module names is discouraged (see https://peps.python.org/pep-0008/#package-and-module-names). Maybe you can find a shorter module name with less/no underscores. Just nitpicking.